<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>formAccountManager</class>
 <widget class="QMainWindow" name="formAccountManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>918</width>
    <height>416</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>11</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>Account Manager</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="2" column="1">
     <widget class="QPushButton" name="btnAdd">
      <property name="font">
       <font>
        <pointsize>11</pointsize>
       </font>
      </property>
      <property name="text">
       <string>Add</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1" colspan="4">
     <widget class="QTableWidget" name="tbAccount">
      <property name="rowCount">
       <number>0</number>
      </property>
      <property name="columnCount">
       <number>6</number>
      </property>
      <attribute name="horizontalHeaderDefaultSectionSize">
       <number>150</number>
      </attribute>
      <attribute name="horizontalHeaderStretchLastSection">
       <bool>true</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <column>
       <property name="text">
        <string>Type</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>User Name</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Password</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Proxy</string>
       </property>
      </column>
      <column>
       <property name="text">
        <string>Profile</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="2" column="4">
     <widget class="QPushButton" name="btnExit">
      <property name="text">
       <string>Exit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="btnEdit">
      <property name="text">
       <string>Edit</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="btnDelete">
      <property name="text">
       <string>Delete</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>btnExit</sender>
   <signal>clicked()</signal>
   <receiver>formAccountManager</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>720</x>
     <y>367</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>207</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>tbAccount</sender>
   <signal>itemDoubleClicked(QTableWidgetItem*)</signal>
   <receiver>btnEdit</receiver>
   <slot>click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>402</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>244</x>
     <y>367</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
